app/uk/gov/hmrc/plasticpackagingtax/registration/views/confirm_address.scala.html

```diff
@@ -61,7 +70,7 @@ <h1 id="heading" class="govuk-heading-l">@messages("primaryContactDetails.confir
                     content = Text(messages("primaryContactDetails.confirmAddress.registeredBusinessAddress"))
                 ),
                 value = Value(
-                    content = HtmlContent(incorporationAddressDetails.toHtml)
+                    content = HtmlContent(extractAddress(businessAddress))
```

Comment:
TIP - not related to this PR.   We had some very similar code in our project that got flagged up in a ZAP test.
Basically, if it is possible for a user to enter "code" as one of their address lines then this code will be rendered by the HtmlContent.

Solution is to restrict the user's input (which might be difficult with addresses) or escape the input when rendering here.  Here is a snip of our similar code - note HtmlFormat.escape(_)
```
            value = Value(
                content = HtmlContent(answers.claimantAddress.map(claimantAddress => Seq(
                    Some(claimantAddress.addressLine1),
                    claimantAddress.addressLine2,
                    Some(claimantAddress.city),
                    Some(claimantAddress.postCode)
                ).flatten.map(HtmlFormat.escape(_)).mkString("<br>")).getOrElse(""))
            ),
```

Reply:
nice, thanks a lot, made that change.